demos/projects/ST/stm32h745i_discovery/cm7/st_code/lwip/system/arch/sys_arch.h

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Do we need all of these checked in here or can we add them as cmake modules / submodule https://github.com/STMicroelectronics/STM32CubeH7 (https://github.com/STMicroelectronics/STM32CubeH7/blob/ccb11556044540590ca6e45056e6b65cdca2deb2/Middlewares/Third_Party/LwIP/system/arch/sys_arch.h for this file)

Reply:
It can be done, but let me do this in next PR